packages/cosmic-proto/dist/agoric/swingset/msgs.d.ts

```diff
@@ -30,15 +30,15 @@ export interface MsgWalletActionResponse {}
 export interface MsgWalletSpendAction {
   owner: Uint8Array;
   /** The action to perform, as JSON-stringified marshalled data. */
-  spendAction: string;
+  spend_action: string;
```

Comment:
That's not backwards compatible.

How about building the stubs this way in a different file or different package?

If we're serious about an in-place change, let's please have an issue and discuss it with all the relevant stakeholders.

Reply:
What consumes the earlier values?

Reply:
I take it back. https://github.com/Agoric/dapp-inter/blob/043f90edf22b30cb78cdca94e6aae1fdf045d27f/src/components/ProvisionSmartWalletDialog.tsx#L30  would break from https://github.com/Agoric/agoric-sdk/pull/8544/files#diff-3502d09501c9eb8d754c1f3332fabe4314472c7ff4ddac54b2fabc645dfef3e9L59

I'll try to find a way to solve the type puzzle without breaking JS exports